### ISSUE TITLE
QDesktopServices can not open URL with unicode path

### ISSUE THREAD

**Issue by akbargumbira** [inasafe/inasafe#1879]
e.g `QDesktopServices.openUrl(QUrl(u'file:///home/akbar/Documents/InaSAFE scenario\xfa'))`
It returns True without opening the directory

We use QDesktopServices in some places in InaSAFE code base

**Comment by lucernae** [comment]
I tested this and it seems that QDesktopServices.openUrl will always return True regardless the file is exists or not. Not just because of the unicode in the Url.

I also try to open unicode path using QDesktopServices, and it actually opens the folder.
Perhaps the bug lies somewhere else?

For example, (because of the wrong filename format)
I tried to create a file using filename : 'scenario\xfa.txt' and QDesktopService can't find it.
But, when I create the file using filename : u'scenario\xfa.txt', QDesktopService is able to find it.
